packages/validator/src/services/indices.ts

```diff
@@ -122,6 +122,9 @@ export class IndicesService {
 
   private async fetchValidatorIndices(pubkeysHex: string[]): Promise<ValidatorIndex[]> {
     const res = await this.api.beacon.getStateValidators("head", {id: pubkeysHex});
+    if (res.status === 404) {
+      this.logger.warn("Validator pubkey not found in state");
```

Comment:
not sure what this would achieve, not we are logging a warning and error which makes it even more noisy